lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_bump_version_beta.rb

```diff
@@ -0,0 +1,71 @@
+module Fastlane
+  module Actions
+    class AndroidBumpVersionBetaAction < Action
+      def self.run(params)
+        UI.message "Bumping app release version..."
+         
+        require_relative '../../helper/android/android_git_helper.rb'
+        require_relative '../../helper/android/android_version_helper.rb'
+
+        Fastlane::Helpers::AndroidGitHelper.check_on_branch("release")
+        create_config()
+        show_config()
+
+        UI.message "Updating gradle.properties..."
```

Comment:
I assume this is a typo? We don't keep our version numbers/codes in `gradle.properties`. They are in `build.gradle`.

Reply:
Yup! It is!!!
Thanks for catching it!